HOWTOADDSERVICE.md

```diff
@@ -9,6 +9,7 @@ The following values are required to add the configuration for a new service.
 | `enrolment`             | The enrolment key for the new service.  Details for creating a new service enrolment key can be found [here](https://github.com/hmrc/service-enrolment-config). | 
 | `shortName`             | This is the abbreviation of the service. | 
 | `callBack`              | ECC will re-direct the user to this url once they have the requested enrolment (see below). | 
+| `accessibility`         | The url of the accessibility statement of the calling service.  This url should be available to users who _are not enrolled_ to the service. | 
```

Comment:
Are the underscores here make any format differences?

Reply:
_italics_

Reply:
Thanks

Reply:
It's not *best* practice though apparently - https://www.markdownguide.org/basic-syntax
So I'll change it.

Reply:
Might even go as far as ***this*** as we really want this point to stand out.